content/get-started/multisig-accounts.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Multisig Accounts
+seo-title: Multisig Accounts
+---
+
+## Create a multisig account
+
+Iron Fish supports multisig accounts using [FROST](https://frost.zfnd.org/frost.html), a threshold signature scheme.
+
+### Create a participant identity
+
+Each signer or participant in an Iron Fish multisig account must first generate an identity
+
+```sh
+ironfish wallet:multisig:participant:create
+```
+
+The participant identity uniquely identifies the participant in the group. The identity is also a public key and allows other members of the group to encrypt data for that participant. The participant's secret key is stored in their Iron Fish wallet.
+
+### Generate multisig account imports
+
+Once all participants have created and shared their identities you can create a multisig account for the list of participant identities using [Trusted Dealer Key Generation](https://frost.zfnd.org/tutorial/trusted-dealer.html). When creating an account the trusted dealer must also specify the minumum number of signers required to sign a transaction.
```

Comment:
I would avoid linking the frost pages: they're very technical, and besides we have introduced some tweaks to frost. People reading the frost pages (and the TDK page in particular) might think that they need to take special precautions, when in practice we have solved most of the problems for them already